src/droplet.F

```diff
@@ -528,8 +533,9 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
     Depart(isw)    = numDepartSW
     Depart(ise)    = numDepartSE
 
-    numDepart=SUM(Depart)
-    if(numDepart .gt. 0) allocate(Depart_ind(numDepart))
+     numDepart=SUM(Depart)
+     if(numDepart .gt. 0) allocate(Depart_ind(max(numDepart,padding)))
```

Comment:
why not just `allocate(Depart_ind(numDepart))`?

Reply:
This is related to the lazy garbage collection and memory leakage in OpenACC.  If the size of the array in this case Depart_ind increases in successive calls then it will allocate a new array that is large enough.  Unfortunately, it does not immediately deallocate the old small array on the GPU device.  So this results in a memory leak until a garbage collection code deallocates all the unused memory.  I therefore choose to allocate a much larger array initially in an attempt to eliminate memory leaks.

Reply:
Oh, I see. Thanks for the clarification. It sounds like `!$acc exit data delete (Depart_ind)` does not work as expected. Probably we can discuss with the NVIDIA team next time.

Reply:
@johnmauff , if I used the `namelist_ASD.verify` namelist, what value of`timax` should I use to trigger the memory leak error without the code modification here?

Reply:
This no longer seems an issue and we have removed the `max` function.